keyboards/duck/tcv3/matrix.c

```diff
@@ -132,7 +140,7 @@ static void init_rows(void) {
 }
 
 static uint8_t read_rows(uint8_t col) {
-  if (col == 20) {
+  if (col == 19) {
```

Comment:
If I had make a guess, this is where the issue actually lies.

Reply:
Keeping that at 20, made the reset key not work in regular mode. I miscounted lol. And yes I was curious about that too, and no unfortunately didn't work either =(.